app/move/controllers/create/base.js

```diff
@@ -24,6 +27,48 @@ class CreateBaseController extends FormWizardController {
 
     next()
   }
+
+  async locals(req, res, callback) {
+    try {
+      const currentLocation = req.session.currentLocation
+      const {
+        date,
+        first_names: firstNames,
+        last_name: lastName,
+        time_due: timeDue,
+        move_type: moveType,
+        to_location: toLocation,
+        additional_information: additionalInfo,
+      } = req.sessionModel.toJSON()
+      let toLocationDetail
+
+      if (toLocation) {
+        toLocationDetail = await referenceDataService.getLocationById(
+          toLocation
+        )
+      }
+
+      const moveSummary = presenters.moveToMetaListComponent({
+        date,
+        time_due: timeDue,
+        move_type: moveType,
+        additional_information: additionalInfo,
+        from_location: currentLocation,
+        to_location: toLocationDetail,
+      })
+      const person = personService.transform({
```

Comment:
I think there is a `person` property on the session model that is stored after the first step. You could pass that straight in here instead?